server/routerlicious/packages/services-utils/src/deleteSummarizedOps.ts

```diff
@@ -21,9 +21,10 @@ export async function deleteSummarizedOps(
             return Promise.reject(error);
         }
 
-        const uniqueDocuments = await documentsCollection.aggregate([
+    const uniqueDocumentsCursor = await documentsCollection.aggregate([
             { $group: { _id: { documentId: "$documentId", tenantId: "$tenantId" } } },
-        ]).toArray();
+    ]);
+    const uniqueDocuments = await uniqueDocumentsCursor.toArray();
```

Comment:
For my own learning: why was this change needed?

Reply:
Yes, I have to introduced a breaking change unfortunately